docs/docs/migration-guide.mdx

```diff
@@ -13,6 +13,35 @@ This page contains information about changes between major versions and
 how you can migrate from one version to another.
 
 
+## Rasa 3.2 to 3.3
+
+### ConcurrentRedisLockStore
+
+The [`ConcurrentRedisLockStore`](./lock-stores.mdx#concurrentredislockstore) is the new Redis implementation of the abstract interface `LockStore`.
+This implementation is able to handle messages received in parallel for the same conversation id by different Rasa instances.
+
+In order to switch from using the `RedisLockStore` to the `ConcurrentRedisLockStore`, specify the parameter (new in 3.3) `concurrent_mode` as `true` in `endpoints.yml`:
+
+```yaml-rasa
+lock_store:
+    type: "redis"
+    concurrent_mode: true
+    url: <url of the redis instance, e.g. localhost>
+    port: <port of your redis instance, usually 6379>
+    password: <password used for authentication>
+    db: <number of your database within redis, e.g. 0>
+    key_prefix: <alphanumeric value to prepend to lock store keys>
+```
+
+Note that not specifying the `concurrent_mode` parameter or setting it to `false` will default to using the `RedisLockStore` implementation.
+
+No database migration is required when switching to the `ConcurrentRedisLockStore`, and you may use the same Redis database number.
+You may want to delete all the preexisting keys if using the same Redis database number.
+The reason for this is because what used to be stored by the `RedisLockStore`(i.e., serialized [TicketLock](https://rasa.com/docs/rasa/reference/rasa/core/lock/#ticketlock-objects) instances) is no longer stored by the new `ConcurrentRedisLockStore`.
```

Comment:
```suggestion
There is no overlap in locks stored when using the `RedisLockStore` and the `ConcurrentRedisLockStore`, because the `RedisLockStore` persists serialized [TicketLock](https://rasa.com/docs/rasa/reference/rasa/core/lock/#ticketlock-objects) instances while
```